test/lib/RRMapParser/FW1886.json

```diff
@@ -0,0 +1 @@
98,30],[27,29],[96,29],[97,29],[27,28],[87,28],[88,28],[96,28],[97,28],[98,28],[27,27],[50,27],[51,27],[52,27],[53,27],[54,27],[55,27],[56,27],[57,27],[58,27],[59,27],[60,27],[61,27],[62,27],[63,27],[64,27],[65,27],[66,27],[67,27],[68,27],[69,27],[70,27],[71,27],[72,27],[73,27],[74,27],[75,27],[76,27],[77,27],[78,27],[79,27],[80,27],[81,27],[82,27],[83,27],[84,27],[85,27],[86,27],[87,27],[88,27],[89,27],[90,27],[91,27],[92,27],[93,27],[94,27],[96,27],[97,27],[98,27],[27,26],[28,26],[29,26],[30,26],[31,26],[32,26],[33,26],[34,26],[35,26],[36,26],[37,26],[38,26],[39,26],[40,26],[41,26],[42,26],[43,26],[44,26],[45,26],[46,26],[47,26],[48,26],[49,26],[50,26],[51,26],[52,26],[58,26],[59,26],[78,26],[79,26],[80,26],[81,26],[82,26],[83,26],[84,26],[85,26],[86,26],[87,26],[88,26],[91,26],[92,26],[93,26],[94,26],[95,26],[96,26],[97,26]]}},"path":{"current_angle":176.3358156792593,"points":[[25600,25600],[25602,25603],[25634,25619],[25663,25630],[25743,25621],[25763,25627],[25789,25578],[25896,25526],[26052,25538],[26126,25542],[26272,25533],[26442,25530],[26570,25524],[26717,25518],[26849,25514],[27021,25508],[27193,25506],[27325,25507],[27327,25509],[27328,25432],[27327,25296],[27322,25301],[27277,25247],[27266,25231],[27217,25235],[27171,25237],[27173,25237],[27113,25272],[26994,25254],[26971,25258],[26884,25209],[26874,25212],[26842,25211],[26654,25212],[26452,25216],[26245,25222],[26047,25220],[25846,25219],[25643,25219],[25437,25227],[25234,25229],[25033,25202],[24828,25197],[24629,25195],[24429,25191],[24224,25187],[24180,25187],[24182,25198],[24178,25353],[24173,25557],[24169,25713],[24169,25694],[24180,25692],[24278,25721],[24284,25724],[24314,25717],[24534,25723],[24664,25731],[24660,25729],[24685,25690],[24759,25586],[24913,25529],[25060,25562],[25095,25587],[25100,25581],[25219,25552],[25342,25587],[25439,25656],[25441,25647],[25473,25642],[25532,25616],[25631,25628],[25654,25627],[25726,25623],[25734,25621],[25753,25518],[25705,25489],[25574,25486],[25371,25487],[25126,25480],[24828,25466],[24518,25456],[24335,25450],[24322,25451],[24323,25449],[24506,25450],[24797,25449],[25094,25453],[25405,25454],[25722,25454],[26030,25450],[26336,25448],[26638,25448],[26920,25448],[27003,25445],[26996,25437],[26921,25445],[26638,25467],[26336,25490],[26033,25509],[25727,25527],[25428,25542],[25115,25558],[25011,25562],[25050,25558],[24995,25523],[24802,25468],[24693,25487],[24639,25537],[24601,25545],[24348,25540],[24261,25539],[24242,25448],[24271,25349],[24289,25325],[24514,25325],[24814,25326],[25118,25331],[25428,25337],[25735,25335],[26047,25332],[26347,25331],[26659,25329],[26880,25328],[26989,25345],[27197,25383],[27240,25389],[27336,25384],[27322,25382],[27332,25322],[27295,25286],[27259,25222],[27274,25257],[27198,25254],[27171,25246],[27133,25281],[27009,25295],[26930,25230],[26921,25219],[26763,25217],[26560,25223],[26359,25229],[26158,25225],[25964,25222],[25761,25220],[25557,25220],[25354,25233]]},"charger":[25602,25948],"robot":[25226,25227]}
```

Comment:
Can you pretty-print this? e.g. using `python -m json.tool`
This would make any future diffs easier to read.

Reply:
done